spec/models/laa_reference_spec.rb

```diff
@@ -109,4 +109,42 @@
       expect(described_class.new(maat_reference: nil)).not_to be_dummy_maat_reference
     end
   end
+
+  describe ".retrieve_by_defendant_id_and_optional_maat_reference" do
+    before { laa_reference.save! }
+
+    context "when only defendant id provided" do
+      subject(:retrieve) { described_class.retrieve_by_defendant_id_and_optional_maat_reference(defendant_id) }
```

Comment:
instead of `retrieve`, should we name it accordinly to the retrieved subject?
maybe `defendant` ?

Reply:
So it's an LAA reference, not a defendant, that we're retrieving. And there's already `let` called `laa_reference` in this spec, so I've gone with `retrieved_laa_reference`